<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditSelectionInterface</class>
 <widget class="QDialog" name="EditSelectionInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1350</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1350</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>260</y>
     <width>1153</width>
     <height>312</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="Button1">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button2">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button3">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button4">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button5">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="Button6">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button7">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button8">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button9">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Button10">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>225</width>
          <height>150</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="ButtonQuit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>650</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
